data/izam-mohammed.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "Izam Mohammed",
+    "type": "personal",
+    "displayStatsPublic": true,
+    "bio": "Open Source Enthusiast | **Data Scientist** | Freeelancer",
+    "avatar": "https://github.com/izam-mohammed.png",
```

Comment:
Our recent update no longer needs the `avatar` key. So please remove this avatar field from your json. 

Updated docs: https://linkfree.eddiehub.io/docs/quickstart
```suggestion
```

Reply:
okey